core/drivers/crypto/hisilicon/hpre_main.c

```diff
@@ -0,0 +1,321 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2024 HiSilicon Limited.
+ * Kunpeng hardware accelerator HPRE module init.
+ */
+#include "hpre_main.h"
+
+/* base config */
+#define HPRE_COMMON_CNT_CLR_CE		0x301000
+#define HPRE_CFG_AXCACHE		0x301010
+#define HPRE_RDCHN_INI_CFG		0x301014
+#define HPRE_BD_ENDIAN			0x301020
+#define HPRE_ECC_BYPASS			0x301024
+#define HPRE_POISON_BYPASS		0x30102c
+#define HPRE_BD_ARUSR_CFG		0x301030
+#define HPRE_BD_AWUSR_CFG		0x301034
+#define HPRE_TYPES_ENB			0x301038
+#define HPRE_DATA_RUSER_CFG		0x30103c
+#define HPRE_DATA_WUSER_CFG		0x301040
+#define HPRE_HAC_INT_MASK		0x301400
+#define HPRE_RAS_ECC_1BIT_TH		0x30140c
+#define HPRE_RAS_CE_ENB			0x301410
+#define HPRE_RAS_NFE_ENB		0x301414
+#define HPRE_RAS_FE_ENB			0x301418
+#define HPRE_HAC_INT_SRC		0x301600
+#define HPRE_RDCHN_INI_ST		0x301a00
+#define HPRE_OOO_SHUTDOWN_SEL		0x301a3c
+#define HPRE_CORE_ENB			0x302004
+#define HPRE_CORE_INI_CFG		0x302020
+#define HPRE_CORE_INI_STATUS		0x302080
+/* clock gate */
+#define HPRE_CLKGATE_CTL		0x301a10
+#define HPRE_PEH_CFG_AUTO_GATE		0x301a2c
+#define HPRE_CLUSTER_DYN_CTL		0x302010
+#define HPRE_CORE_SHB_CFG		0x302088
+#define HPRE_CORE_GATE_ENABLE		GENMASK_32(31, 30)
+
+#define HPRE_AXCACHE_MASK		0xff
+#define HPRE_HAC_INT_DISABLE		0x1ffffff
+#define HPRE_RAS_CE_MASK		0x1
+#define HPRE_RAS_NFE_MASK		0x1fffffe
+#define HPRE_RAS_FE_MASK		0
+#define HPRE_BD_LITTLE_ENDIAN		0
+#define HPRE_RSA_ENB		BIT(0)
+#define HPRE_ECC_ENB		BIT(1)
```

Comment:
Could you add another indentation tab in the 2 lines above?

Reply:
OK，It seems all answer and comments are solved, the PR is ready for another round of review.